src/operations/insert.ts

```diff
@@ -105,62 +103,5 @@ export interface InsertManyResult<TSchema = Document> {
   insertedIds: { [key: number]: InferIdType<TSchema> };
 }
 
-/** @internal */
-export class InsertManyOperation extends AbstractOperation<InsertManyResult> {
```

Comment:
Now that this operation is gone, maybe we can clean up the method `maybeAddIdToDocuments` in utils? I don't think we have any place remaining where it would take an array of docs.